crates/concrete-syntax/src/models/concrete_syntax/interpreter.rs

```diff
@@ -470,6 +483,7 @@ fn create_empty_captured_node() -> CapturedNode {
       end_point: crate::models::matches::Point { row: 0, column: 0 },
     },
     text: String::new(),
+    node_type: "empty".to_string(),
```

Comment:
Perhaps make this more verbose like `<empty-node-type-that-should-never-be-matched>`?

`empty` feels too generic and if we see this when debugging we might think it's a valid node type.